src/pages/iou/request/IOURequestStartPage.tsx

```diff
@@ -266,7 +266,7 @@ function IOURequestStartPage({
                                         </TabScreenWithFocusTrapWrapper>
                                     )}
                                 </TopTab.Screen>
-                                {!manualDistanceTrackingEnabled && (
+                                {(!manualDistanceTrackingEnabled || iouType === CONST.IOU.TYPE.SPLIT) && (
```

Comment:
I think this change is not needed since we want to keep the split expense as it is.